doc/development/plugins.rst

```diff
@@ -58,6 +58,12 @@ and subclassing it:
     which contains a lot of functionality specific to computations based on qubits. We will
     take a deeper look at this important case below.
 
+.. warning::
+
+    The API of :class:`~.DefaultQubit` will be updated soon to follow a new interface
+    defined by the experimental :class:`pennylane.devices.experimental.Device` class. To view the
+    exact changes coming to :class:`~.DefaultQubit`, see :class:`pennylane.devices.experimental.DefaultQubit2`.
```

Comment:
```suggestion
    The API of PennyLane devices will be updated soon to follow a new interface defined by
    the experimental :class:`pennylane.devices.experimental.Device` class. This guide describes
    how to create a device with the current :class:`pennylane.Device` and
    :class:`pennylane.QubitDevice` base classes, and will be updated as we switch to the new API.
    In the meantime, please reach out to the PennyLane team if you would like help with building
    a plugin, either by creating an [issue](https://github.com/PennyLaneAI/pennylane/issues) or
    by posting in our [discussion forum](https://discuss.pennylane.ai/).
```
For this particular page, we need to be a bit more general than for default.qubit.